app/uk/gov/hmrc/cataloguefrontend/whatsrunningwhere/model.scala

```diff
@@ -78,14 +79,43 @@ object JsonCodecs {
   val profileFormat: OFormat[Profile] = {
     implicit val ptf = profileTypeFormat
     implicit val pnf = profileNameFormat
-    ( (__ \ "type").format[ProfileType]
+    ((__ \ "type").format[ProfileType]
     ~ (__ \ "name").format[ProfileName]
-    )(Profile.apply, unlift(Profile.unapply))
+    ) (Profile.apply, unlift(Profile.unapply))
+  }
+
+  val deploymentEventFormat: Format[DeploymentEvent] = {
+    implicit val vnf = versionNumberFormat
+    implicit val tsf = timeSeenFormat
+    implicit val dsf = deploymentStatusFormat
+
+    ( (__ \ "deploymentId").format[String]
+    ~ (__ \ "status").format[DeploymentStatus]
+    ~ (__ \ "version").format[VersionNumber]
+    ~ (__ \ "time").format[TimeSeen]
+    )(DeploymentEvent.apply, unlift(DeploymentEvent.unapply))
+  }
+
+  val serviceDeploymentsFormat: Format[ServiceDeployments] = {
+    implicit val devf = deploymentEventFormat
+    implicit val anf  = applicationNameFormat
+    implicit val enf  = environmentFormat
+
+    ( (__ \ "serviceName").format[ApplicationName]
+    ~ (__ \ "environment").format[Environment]
+    ~ (__ \ "deploymentEvents").format[Seq[DeploymentEvent]]
+    ~ (__ \ "lastCompleted").formatNullable[DeploymentEvent]
+    )(ServiceDeployments.apply, unlift(ServiceDeployments.unapply))
   }
 }
 
 case class TimeSeen(time: LocalDateTime)
 
+object TimeSeen {
+  implicit val localDateOrdering: Ordering[LocalDateTime] = Ordering.by(_.toEpochSecond(ZoneOffset.UTC))
```

Comment:
`localDateOrdering` should only be implicit in the scope of creating `timeSeenOrdering` - TimeSeen should not be introducing an implicit LocalDateTime.

Reply:
Have pushed a small alteration to address this.